lib/developer_portal/app/views/shared/_swagger.html.erb

```diff
@@ -13,15 +13,24 @@
   <p><a href='#'>Sign in</a> to your account for quick access to useful values.</p>
 </div>
 
+<% url_opts = { format: :json }; url_opts[:services] = services.pluck(:system_name).join(',') unless services.empty? %>
+
 <script type="text/javascript">
   $(function () {
-    var url = "<%= services.empty? ? swagger_spec_index_url(format: :json) : swagger_spec_index_url(format: :json, services: services.join(',')) %>";
+    var host = "<%= @api_docs_service.host_with_port %>";
+    var spec = <%= ActionController::Base.helpers.sanitize(@api_docs_service.body) %>;
```

Comment:
Same as https://github.com/3scale/porta/pull/539/files#r257965975
Whatever we decide for there, we should do the same here.